src/Core/Models/Business/OrganizationUserInvite.cs

```diff
@@ -0,0 +1,44 @@
+using System.Collections.Generic;
+using Bit.Core.Models.Api;
+using Bit.Core.Models.Interfaces;
+using System.Linq;
+using Bit.Core.Models.Data;
+
+namespace Bit.Core.Models.Business
+{
+    public class OrganizationUserInvite: IPermissions
```

Comment:
Spacing before the colon, ` : `

Reply:
Not applicable with IPermissions getting the boot